exoctk/exoctk_app/app_exoctk.py

```diff
@@ -384,6 +384,12 @@ def groups_integrations_results():
         params['filt_ta'] = params['{}_filt_ta'.format(ins)]
         params['subarray'] = params['{}_subarray'.format(ins)]
         params['subarray_ta'] = params['{}_subarray_ta'.format(ins)]
+        
+        # Convert the obs_time to hours
+        if params['time_unit'] != 'hours':
+            params['obs_time'] = params['obs_time']*24
+            params['time_unit'] = 'days'
```

Comment:
But didn't you just convert it to hours in line 390? Shouldn't line 391 be `params['time_unit'] = 'hours'`

Reply:
After our in person chat, I changed it 'hours'!